packages/synapse-react-client/test/lib/containers/oauth/CreateOAuthClient.integration.test.tsx

```diff
@@ -42,6 +42,10 @@ const defaultProps: CreateOAuthModalProps = {
   setIsShowingConfirmModal: jest.fn(),
 }
 
+jest
+  .spyOn(SynapseClient, 'isOAuthClientReverificationRequired')
+  .mockReturnValue(Promise.resolve({ reverificationRequired: true }))
```

Comment:
Move mock outside of specific test, so the mock applies in all cases, e.g. in "Shows a warning modal when deleteing a client" as well as "Shows a warning modal when changing redirect uri"